00_Base/src/interfaces/repository.ts

```diff
@@ -62,7 +62,6 @@ export abstract class CrudRepository<T> extends EventEmitter {
     this.emit('created', [result]);
```

Comment:
Minor: How about using /* eslint-disable @typescript-eslint/member-ordering */? In this case, we don't need to change the existed code and methods for the same db operation can be stay together.

Reply:
Could you please also help fix the warning, Forbidden non-null assertion in the `Authorization`? The fix can be like
```
if (updatedValue !== undefined && savedIdTokenInfo) {
      // Null can still be used to remove data
     savedIdTokenInfo.setDataValue(k, valueIdTokenInfo.getDataValue(k));
}
```
Because `savedIdTokenInfo!` means that we are sure `savedIdTokenInfo` is always not null, then add undefine/null check should not make difference.

Reply:
I am open to ignoring the eslint rule here. Personally, I would prefer a repetitive structure which eslint enforces. Even if the underlying `_query` belong together, when looking at the code, you have it in an IDE, so you can just click jump to the impl. 

In regards to the non-null assertion in `Authorization` I was a bit hesitant due to the commented line `// Null can still be used to remove data` which I interpret as we want to explicitly allow null here. So if we add a null check, we will not be allowing null to execute. I don't have the full context on functionality, as I was just fixing minor errors and didn't take the time to deep dive into this section. @thanaParis any thoughts?

Reply:
> I am open to ignoring the eslint rule here. Personally, I would prefer a repetitive structure which eslint enforces. Even if the underlying _query belong together, when looking at the code, you have it in an IDE, so you can just click jump to the impl.

I could go either way on this. If more people feel the eslint method of organization is better, we should change it.

> In regards to the non-null assertion in Authorization I was a bit hesitant due to the commented line // Null can still be used to remove data which I interpret as we want to explicitly allow null here.

Yes, if a field has been removed vs a field was not included, the first is null the second is undefined. So explicitly allowing null is important here.

Reply:
The people have spoken. We will be going stricter and try to automate these things :)